src/Compilers/CSharp/Portable/Symbols/Source/SourceEnumConstantSymbol.cs

```diff
@@ -198,7 +200,9 @@ protected override ConstantValue MakeConstantValue(HashSet<SourceFieldSymbolWith
                 {
                     return Microsoft.CodeAnalysis.ConstantValue.Unset;
                 }
+#nullable disable // Can 'otherValue' be null? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
I didn't understand why there would be a warning here. `GetConstantValue` seems to be returning an oblivious return value at the moment.
I believe those methods cannot return null (should return `ConstantValue.Unset` instead), but probably need to annotate them to help confirm.

Reply:
Hum, looking in `SourceMemberFieldSymbol`, I see that we're checking on possible `null` returned from `GetConstantValue`, so that makes me doubt about my above statement... Maybe `GetConstantValue` could return `null` after all?
Do you want to try and annotate that method in this PR?


``` chsarp
        internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<SynthesizedAttributeData>? attributes)
        {
            base.AddSynthesizedAttributes(moduleBuilder, ref attributes);

            var compilation = this.DeclaringCompilation;
            var value = this.GetConstantValue(ConstantFieldsInProgress.Empty, earlyDecodingWellKnownAttributes: false);

            // Synthesize DecimalConstantAttribute when the default value is of type decimal
            if (this.IsConst && value != null
                && this.Type.SpecialType == SpecialType.System_Decimal)
            {
```